ossfuzz/standaloneengine.cc

```diff
@@ -0,0 +1,74 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "testinput.h"
+
+/**
+ * Main procedure for standalone fuzzing engine.
+ *
+ * Reads filenames from the argument array. For each filename, read the file
+ * into memory and then call the fuzzing interface with the data.
+ */
+int main(int argc, char **argv)
```

Comment:
I think we'll want to take the one from zstd, since that works recursively on directories. Then we can easily run regression tests on CI.

But that can be done later.

Reply:
Happy to use whatever you want :smile: though maybe would be appropriate in a followup commit?

Reply:
Yeah, we can do it later.